.github/workflows/a11y-add-alt-bot.yml

```diff
@@ -0,0 +1,26 @@
+jobs:
+  accessibility_alt_text_bot:
+    if: ${{ github.event.issue || github.event.pull_request || github.event.discussion }}
+    name: Check alt text is set on issue or pull requests
```

Comment:
[Cleanup] This repo generally omits `name` when there's not a strong need for it. It's a bit less explicit/verbose than the suggested blurb from the bot's README.

```suggestion
```